docs/onr/submitter_guide.md

```diff
@@ -0,0 +1,90 @@
+# ONM Research Initiative — Submitter’s Guide
+
+Welcome to the ONM Research Initiative! This guide outlines how to submit your work for review and inclusion in the ONM Community Peer Review Program.
+
+#### Submissions are accepted on a rolling basis
+
+---
+
+## What Can You Submit?
```

Comment:
Maybe we could add an initial review step, where we do a quick review of the submission to see if it is aligned with the criteria. Let's say for example some component doesn't have a license defined or something - that we would reject and point them to the reasons (and invite them to resubmit)